tests/testthat/_snaps/conversion.md

```diff
@@ -41,14 +41,14 @@
     Code
       as_adjacency_matrix(g, attr = "bla", sparse = FALSE)
     Condition
-      Error in `get.adjacency.dense()`:
+      Error in `get.adjacency.sparse()`:
```

Comment:
Can we change this to `Error in as_adjacency_matrix()` ?

Reply:
Yes, I added `call. = FALSE` to deep `stop()` calls for better error reporting.